.github/workflows/test.yml

```diff
@@ -1,12 +1,19 @@
 name: Test
 
-on: push
+on:
+  pull_request:
+  push:
+    branches:
+      - develop
+      - main
 
 env:
+  DANGER_GITHUB_API_TOKEN: ${{ github.token }}
   DOCKER_REGISTRY_HOST: ${{ secrets.DOCKER_REGISTRY_HOST }}
   DOCKER_REGISTRY_USERNAME: ${{ github.repository_owner }}
   DOCKER_REGISTRY_TOKEN: ${{ secrets.DOCKER_GITHUB_TOKEN }}
   DOCKER_IMAGE: ${{ github.repository }}
+  NODE_VERSION: 16
```

Comment:
@aetheryna Should we use the same version as what you have in the `.tool-version`?

Reply:
Yeah your right, I can see why you'd suggest this as well. Good spot! 👍🏾 Updated in [28acf62](https://github.com/aetheryna/google_scraper_ruby/pull/26/commits/28acf62d687142beca1e501024f8f6caa7644225)